tst/unit/test_sparse_pack.cpp

```diff
@@ -128,10 +128,10 @@ TEST_CASE("Test behavior of sparse packs", "[SparsePack]") {
            "blocks") {
         // Create a pack use type variables
         auto sparse_pack =
-            parthenon::SparsePack<v5, v3>::Get(&mesh_data, {Metadata::WithFluxes});
+            parthenon::SparsePack<v5, v3>::Get<MeshData<Real>, Real>(&mesh_data, {Metadata::WithFluxes});
```

Comment:
It would be great to *not* have to be explicit about `MeshData<Real>` in the template parameter list here but I couldn't immediately figure out how to do it.